tests/services/test_strategy_marketplace_integration.py

```
@@ -84,14 +87,52 @@ async def _reset_database() -> None:
+            for raw_key in keys:
+                key = (
+                    raw_key.decode("utf-8")
+                    if isinstance(raw_key, (bytes, bytearray))
+                    else str(raw_key)
+                )
+                if key in self.store:
+                    del self.store[key]
+                    removed += 1
+            return removed
+
+        async def scan_iter(self, match: str | None = None, count: int | None = None):
```

Ruff (0.13.1)

Unused method argument: `count`

(ARG002)